scripts/api_rst_coverage.py

```diff
@@ -61,21 +63,25 @@ def add_notes(x):
     # class members
     class_members = set()
     for cls in classes:
-        class_members.update([cls.__name__ + '.' + x[0] for x in inspect.getmembers(cls)])
+        class_members.update('%s.%s' % (cls.__name__, x[0])
```

Comment:
We're moving away from this string formatting mechanism in favor of named `kwargs`.